client/lib/domains/index.js

```diff
@@ -22,26 +23,26 @@ function ValidationError( code ) {
 
 inherits( ValidationError, Error );
 
-function canAddGoogleApps( domain ) {
-	var tld = domain.split( '.' )[ 1 ],
-		includesBannedPhrase = some( GOOGLE_APPS_BANNED_PHRASES, function( phrase ) {
-			return includes( domain, phrase );
-		} );
-
-	if ( includes( GOOGLE_APPS_INVALID_TLDS, tld ) || includesBannedPhrase ) {
-		return false;
+function canAddGoogleApps( domainName ) {
+	let domainParts = domainName.split( '.' );
+	if ( domainParts.length !== 2 ) {
```

Comment:
What about [second-level domains](https://en.wikipedia.org/wiki/Second-level_domain), like `co.uk` or `com.br`? I think it's totally possible someone would map their blog to such a domain and try to add Google Apps to it.